src/main/java/com/trzewik/blackjack/deck/Deck.java

```diff
@@ -8,30 +8,30 @@
 
 public class Deck {
 
-    private ArrayList<Card> listOfCards = new ArrayList<>();
+    private ArrayList<Card> Cards = new ArrayList<>();
```

Comment:
to nie jest nazwa klasy.... w Javie tylko nazwy klas piszemy wielka litera, nazwy zmiennych małą